reconcile/utils/external_resource_spec.py

```diff
@@ -116,6 +116,13 @@ def namespace_name(self) -> str:
     def cluster_name(self) -> str:
         return self.namespace["cluster"]["name"]
 
+    @property
+    def environment_type(self) -> str:
+        try:
+            return json.loads(self.namespace["environment"]["labels"])["type"]
+        except KeyError:
+            return "production"
```

Comment:
this exception is a little confusing, `namespace` should always have `environment`, and `labels` now is a required field, so no exception expected in `json.loads(self.namespace["environment"]["labels"])`, then just `type` can be missing.

Should it just be

```
    labels = json.loads(self.namespace["environment"]["labels"])
    return labels.get("type", "production")
```

Reply:
👍 changed in [668e844](https://github.com/app-sre/qontract-reconcile/pull/4827/commits/668e8443ea6792bc49737ddabadd8c7d548c8b9d)